xapian-letor/ranker/ranker.cc

```diff
@@ -354,9 +354,6 @@ Ranker::labelcomparer(const FeatureVector & firstfv, const FeatureVector& second
 std::vector<Xapian::docid>
 Ranker::rank(const Xapian::MSet & mset, const string & model_key, const Xapian::FeatureList & flist)
 {
-    if (mset.empty()) {
-	throw Xapian::InvalidArgumentError("Empty MSet. No documents could be retrieved with the given Query.\n");
-    }
```

Comment:
I left it here because checking for an empty MSet in #156 ..... and resolving it there would be better because that would be the final function instead of this.